src/app/InteractionModelEngine.h

```diff
@@ -234,17 +234,18 @@ class InteractionModelEngine : public Messaging::UnsolicitedMessageHandler,
 
 #if CHIP_CONFIG_ENABLE_READ_CLIENT
     /**
-     *  Activate the idle subscriptions.
      *
-     *  See ReadClient::OnActiveModeNotification
+     *  When a peer becomes active, it sends a check-in message because aMonitoredSubject does not have a subscription to it.
+     * Upon receiving this check-in message, the Client attempts to run the re-subscription process via OnActiveModeNotification
```

Comment:
```suggestion
     *  Notification that aPeer has sent a check-in message because aMonitoredSubject does
     *  not have a subscription to it.
```